Hearthstone Deck Tracker/Windows/OverlayWindow.Tooltips.cs

```diff
@@ -313,7 +313,7 @@ private void SetFlavorTextEntity(Entity entity)
 		{
 			try
 			{
-				if(!Config.Instance.ShowFlavorText || entity?.Card == null)
+				if(!Config.Instance.ShowFlavorText || entity?.Card?.FormattedFlavorText == "")
```

Comment:
You need to use `string.IsNullOrEmpty` here. If entity or card are null this will return false.